packages/shared/components/modals/ProfileActions.svelte

```diff
@@ -56,6 +67,9 @@
                 <span class="text-12 leading-100 text-center text-white uppercase">{profileInitial}</span>
             </div>
             <Text>{profileName}</Text>
+            {#if isLedgerProfile}
```

Comment:
`isLedgerProfile` is a store, so this if will always be true because you are not subscribing to it but checking if the observable store exists, which does
```suggestion
            {#if $isLedgerProfile}
```